config/packages/security.yaml

```diff
@@ -10,7 +10,8 @@ security:
         sylius_shop_user_provider:
             id: sylius.shop_user_provider.email_or_name_based
     encoders:
-        Sylius\Component\User\Model\UserInterface: sha512
+        sylius_default_password_encoder:
```

Comment:
I would name it `sha512` (`default` looks tricky and might break the application if someone suddenly changes the algorithm).

Reply:
Actually, I would leave the encoder name as is and change the service ID of the "default" encoder. So the default encoder is configured to be the sha512 one